<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singlemotion</class>
 <widget class="QWidget" name="singlemotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_down">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下降</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_location">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>钻孔</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Motion_Number">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drilling">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>定位</string>
   </property>
  </widget>
  <widget class="QLabel" name="Drill_motion">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>钻头电机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_up">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>上升</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Motion_Number</tabstop>
  <tabstop>pushButton_drilling</tabstop>
  <tabstop>pushButton_up</tabstop>
  <tabstop>pushButton_location</tabstop>
  <tabstop>pushButton_down</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
